docs/roadmap/cosmos-hub-roadmap-2.0.md

```diff
@@ -52,39 +52,18 @@ The upgrades aim to add features such as liquidity, economic security, usability
 - Gaia v8.0.x
 - Cosmos SDK v0.45
   - Minimal update with small fixes
-- Interchain Security - Required Participation of Provider Chain Validators
-  - The Cosmos solution to shared security that uses IBC Cross Chain Validation (CCV) to relay validator set composition from a Provider Chain (Cosmos Hub) to a Consumer Chain. This validator set is in charge of producing blocks on both networks using separate nodes. Misbehavior on the Consumer Chain results in slashing Provider Chain staking tokens (ATOM).
+- IBC Msg Whitelist to skip MinFee in CheckTX
 - Global Fee Module
   - Allows denoms and min-fees to be governance parameters so gas can be paid in various denoms.
   - Visible on [tgrade](https://github.com/confio/tgrade/tree/main/x/globalfee) already and enabled in [ante.go](https://github.com/confio/tgrade/blob/main/app/ante.go#L72-L92)
```

Comment:
Might want to link instead or additionally to Gaia docs

Reply:
Added